pipeline-scripts/buildlib.groovy

```diff
@@ -78,7 +78,9 @@ def initialize_enterprise_images_dir() {
 def oit( cmd ){
     cmd = cmd.replaceAll( '\n', ' ' ) // Allow newlines in command for readability, but don't let them flow into the sh
     cmd = cmd.replaceAll( ' \\ ', ' ' ) // If caller included line continuation characters, remove them
-    sh "${env.ENTERPRISE_IMAGES_DIR}/oit/oit.py --user=ocp-build --metadata-dir ${env.ENTERPRISE_IMAGES_DIR} ${cmd.trim()}"
+    return sh(
```

Comment:
suggesting capture=false as a default argument. The output of oit is usually useful in the Jenkins log, so I'd like it to continue streaming there by default. 
`return sh( returnStdout: capture, "......")`

Reply:
Updated.